main.py

```diff
@@ -44,6 +44,8 @@
 current_channel: Optional[TextChannel] = None
 # The media in the current channel
 current_channel_content: Optional[List] = None
```

Comment:
And while we're here.

```suggestion
current_channel_content: Optional[List[Tuple[Message, Attachment, str]]] = None
```